src/core/src/op/convert.cpp

```diff
@@ -232,6 +233,7 @@ bool Convert::has_evaluate() const {
         case element::u64:
         case element::f8e4m3:
         case element::f8e5m2:
+        case element::f8e8m0:
```

Comment:
Conversion to this float should be limited to  `f8e2m0` <-> `f16`, `bf16`, `f32`. 
Now the conversion will be generated for other types which will increase bin size.

Reply:
done